roles/openshift_aws/defaults/main.yml

```diff
@@ -154,8 +154,9 @@ openshift_aws_master_group:
   group: master
   tags:
     host-type: master
-    sub-host-type: master
+    sub-host-type: default
```

Comment:
I copied the gcp concept to the aws playbooks to make them functional in https://github.com/openshift/openshift-ansible/pull/8697. I think it'd be fine to keep the config name in a tag but I'm not sure that we need to? Is this `default` sub-host type used for anything in particular? I changed that in the linked PR so that I could differentiate between sub host types.